symplyphysics/laws/thermodynamics/heat_transfer/solution_with_zero_temperature_boundaries.py

```diff
@@ -62,6 +63,13 @@ def calculate_temperature(
     position_: Quantity,
     time_: Quantity,
 ) -> Quantity:
+    if maximum_position_.scale_factor <= 0:
+        raise ValueError("maximum position must be positive")
+    if position_.scale_factor < 0:
+        raise ValueError("position must be non-negative")
+    if position_.scale_factor >= maximum_position_.scale_factor:
```

Comment:
Why position cannot be maximum position?

Reply:
You're right, it can be